api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -49,7 +50,11 @@ trait ValidationErrorConverter {
 
     val jsVals: Seq[(String, JsValue)] = fieldNames.map { fieldName =>
       val row = relevantRow(err, vs)
-      val fieldValue = row.valueOf(fieldName)
+      val fieldValue = if (fieldName == "Metropolitan Statistical Area / Metropolitan Division Name") {
+        CbsaLookup.nameFor(row.valueOf("Metropolitan Statistical Area / Metropolitan Division").toString)
```

Comment:
This needs a spec to show that it produces the correct JSON response for a Q029 failure.